components/FileUploader/index.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    class="w-[358px] h-[165px] rounded-lg bg-base-250 relative border-dashed border-2 border-dimound flex items-center justify-evenly flex-col text-xs text-base-500">
+    class="w-[358px] h-[165px] rounded-lg bg-base-250 relative border-dashed border-2 border-primary flex items-center justify-evenly flex-col text-xs text-base-500">
```

Comment:
In the next update, it is necessary to change pixel sizes to rem units.

Reply:
Of course.